src/Notifications/RefundConfirmation.php

```diff
@@ -58,7 +58,7 @@ public function toMail($notifiable)
         }
 
         if ($refund->isVoucher()) {
-            $message->line("Refund voucher code: " . $this->voucher->code);
+            $message->line("Refund voucher code: " . $this->refund->voucher->code);
```

Comment:
@drewroberts We should refer to `voucher` from `refund` property, right? Since we don't have `voucher` property, so `$this->voucher` will throw an error.

Reply:
I think so, but this will be part of our discussion the next few days with services and package boundaries due to limited testing capabilities.

CC: @sl0wik @pdbreen

Reply:
At this stage, this is a bug fix -- `$this->voucher` doesn't exist, so it needs to be `$this->refund->voucher->code`.